places/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,52 @@
+class UsersController < ApplicationController
+    
+    def index
+        @users = User.all
+    end
+    
+    def show
+        @user = User.find(params[:id])
+        @places = Place.all
+    end
+    
+    def new
+        @user = User.new
+    end
+    
+    def create
+        @user = User.new
+        @user.first_name = params[:user][:first_name]
+        @user.last_name = params[:user][:last_name]
+        @user.email = params[:user][:email]
+        @user.picture = params[:user][:picture]
+        @user.save
```

Comment:
Once your form submits your params should pass in the `place_id`. You could use the same code you have in `add_place` method to associate a place with a user.